test/kanbanize_test.rb

```diff
@@ -46,7 +46,7 @@ def test_push_with_issue_tracking
     @stubs.post url do |env|
       assert_equal 'testdomain.kanbanize.com', env[:url].host
       assert_equal '/index.php/api/kanbanize/git_hub_event', env[:url].request_uri
-      assert_equal %({"ref":"refs/heads/mybranch2"}), env[:body]
```

Comment:
I created two tests because the cases are different.
For the push case, we check that when a branch filter is applied the test will pass for the provided repo ref.
For the issue event we don't provide a "ref" property in the payload(which was the reason the service was failing) and I check this in a separate test case.